### PR TITLE
fix: drag and drop files mime types

### PR DESCRIPTION
**Pull request by yoavrozner**

### CODE REVIEW

File: src/utils/drop.js [hunk 1]
```diff
@@ -107,6 +108,13 @@ async function getEntries(entry, parent, isFirstFolder) {
     }
 }
 
-function getFile(fileEntry) {
-    return new Promise((resolve, reject) => fileEntry.file(resolve, reject));
+async function getFile(fileEntry) {
```
Comment (nettashviro) [inline src/utils/drop.js:111]: Please add docs

**Review by nettashviro — APPROVED** [review]

**Comment by nettashviro** [comment]
Love that you've asked Yosef for pr

File: src/utils/drop.js (hunk 1 above)
Comment (yoavrozner) [reply src/utils/drop.js:111]: Good idea

Comment (Shahar-Y) [reply src/utils/drop.js:109]: Wa sthe documentation added?

**Review by Shahar-Y — CHANGES_REQUESTED** [review]

**Review by Shahar-Y — APPROVED** [review]

**Pull request merged**